mmv1/third_party/terraform/website/docs/d/oracle_database_db_nodes.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Oracle Database"
+description: |-
+  List all database nodes of a Cloud VmCluster.
+---
+
+# google_oracle_database_db_nodes
+
+List all DbNodes of a Cloud VmCluster.
+
+For more information see the
+[API](https://cloud.google.com/oracle/database/docs/reference/rest/v1/projects.locations.cloudVmClusters.dbNodes).
+
+## Example Usage
+
+```hcl
+data "google_oracle_database_db_nodes" "my_db_nodes"{
+	location = "us-east4"
+	cloud_vm_cluster = "vmcluster-id"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `cloud_vm_cluster` - (Required) The ID of the VM Cluster.
+
+* `location` - (Required) The location of the resource.
+
+- - -
+* `project` - (Optional) The project in which the resource belongs. If it
+    is not provided, the provider project is used.
+
+* `db_nodes` - (Output) List of dbNodes. Structure is [documented below](#nested_dbnodes).
```

Comment:
this is closer to "standard" for datasources. For example, see https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/bigquery_table_iam_policy

```suggestion
## Attributes reference

The following attributes are exported:

* `db_nodes` - List of dbNodes. Structure is [documented below](#nested_dbnodes).
```